client/src/app/core/repositories/motions/amendment-repository.service.ts

```diff
@@ -0,0 +1,120 @@
+import { Injectable } from '@angular/core';
+
+import { AmendmentAction } from 'app/core/actions/amendment-action';
+import { MotionAction } from 'app/core/actions/motion-action';
+import { Identifiable } from 'app/shared/models/base/identifiable';
+import { Motion } from 'app/shared/models/motions/motion';
+import { createAgendaItem } from 'app/shared/utils/create-agenda-item';
+import { ViewMotion } from 'app/site/motions/models/view-motion';
+import { BaseRepositoryWithActiveMeeting } from '../base-repository-with-active-meeting';
+import { MotionRepositoryService } from './motion-repository.service';
+import { RepositoryServiceCollector } from '../repository-service-collector';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AmendmentRepositoryService extends BaseRepositoryWithActiveMeeting<ViewMotion, Motion> {
```

Comment:
Please do not create repositories that do not have appropriate models! I need more info why you did it - please give me a call.